packages/sublime_text.rb

```diff
@@ -19,6 +19,7 @@ class Sublime_text < Package
   })
 
   depends_on 'sommelier'
+  depends_on 'gtk2'
```

Comment:
Can you please move this above `depends_on 'sommelier'`?  The reason is there are post install instructions that would be missed.

Reply:
Done